metadata.rb

```diff
@@ -1,10 +1,10 @@
 name             'nsis'
 maintainer       'Ketan Padegaonkar'
 maintainer_email 'ketanpadegaonkar@gmail.com'
-license          'MIT'
+license          'Apache v2.0'
```

Comment:
I got that value from the docs here: https://docs.chef.io/cookbook_repo.html

It said "Apache v2.0". :)